pages/src/App.js

```diff
@@ -48,6 +48,32 @@ class App extends Component {
 
   _onEditTodo = this._onCreateTodo
 
+  _onDeleteElementToDo = id => {
+
+  fetch(`/todos/${id}`, {
+
+    body: JSON.stringify(id),
```

Comment:
Why do you need to send an id?

Reply:
to know wich element we want to remove it

Reply:
The id is already in the URL.

Reply:
Yes, so I don't need to send it.